### PR TITLE
perkeep: unstable-2020-03-23 -> 0.11

### PR DESCRIPTION
**Pull request by danderson**
Signed-off-by: David Anderson <dave@natulte.net>

###### Motivation for this change

New upstream release.

###### Things done

Note I trimmed the produced binaries down a little, to just the client and server tools rather than also include the demo and development binaries.

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by tomberek** [comment]
Result of `nixpkgs-review pr 111262` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>perkeep</li>
  </ul>
</details>

**Review by tomberek — APPROVED** [review]

File: pkgs/applications/misc/perkeep/default.nix [hunk 1]
```diff
@@ -11,41 +11,50 @@ let
...
+    rm -f $out/bin/genfileembed
   '';
 
   meta = with lib; {
     description = "A way of storing, syncing, sharing, modelling and backing up content (née Camlistore)";
     homepage = "https://perkeep.org";
     license = licenses.asl20;
-    maintainers = with maintainers; [ cstrahan kalbasit ];
+    maintainers = with maintainers; [ cstrahan danderson kalbasit ];
     platforms = platforms.unix;
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/perkeep/default.nix:58]:
```suggestion
```

set by buildGoModule

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 111262` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
1 package built:
</summary>
<ul>
<li>perkeep</li>
</ul>
</details>

**Comment by danderson** [comment]
My bad, I saw approval above and got trigger happy. PTAL.

**Comment by bennyandresen** [comment]
I would prefer if the commits were squashed or the `platform` line received a proper commit subject. Then I would approve and set the status to needs_merger. Thanks!

**Pull request merged**